frontend/src/components/Dropdown/SingleSelect/SingleSelect.stories.tsx

```diff
@@ -0,0 +1,182 @@
+import { useCallback, useMemo } from 'react'
+import { Controller, useForm } from 'react-hook-form'
+import { BiHeading, BiRadioCircleMarked } from 'react-icons/bi'
+import { FormControl } from '@chakra-ui/react'
+import { useArgs } from '@storybook/client-api'
+import { Meta, Story } from '@storybook/react'
+
+import Button from '~components/Button'
+import FormErrorMessage from '~components/FormControl/FormErrorMessage'
+import FormLabel from '~components/FormControl/FormLabel'
+
+import { ComboboxItem } from '../types'
+import { itemToLabelString, itemToValue } from '../utils/itemUtils'
+
+import { SingleSelect, SingleSelectProps } from './SingleSelect'
+
+const INITIAL_COMBOBOX_ITEMS: ComboboxItem[] = [
```

Comment:
might be good to add some values with multiple words just to be able to see the difference / catch any edge cases in future. maybe some with a long string as text to see how it behaves

Reply:
Done. Caught a bug where label instead of value is being used to fill the dropdown, so good call to check. 
(We won't use differing labels and values cause it'll be confusing to the users la but haha)